### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by joefozmans** [compsoc-edinburgh/website2016#6]
See the README.md
- [ ] Prerequisites and install instructions
- [ ] Document how to run sans-Docker
- [ ] Run on OS X / Windows

**Comment by hreeder** [comment]
I'd say the first and last items there should be for both docker and non-docker.

**Comment by joefozmans** [comment]
yeah. With regards to Windows, can we just ask people to run bash for windows? :P

**Comment by hreeder** [comment]
That may be the most suitable thing to be honest.

Once Kronk is set up, if it's just post / page creation & population then you shouldn't really need to run a copy locally anyway.

Docker is nice and easy. What's in preview.sh right now will work on docker-machine pretty well, the main thing is I don't believe it will automatically regenerate because of how folder sharing works on docker-machine? It also won't be at http://localhost:4000/, but the IP of the docker machine.
